pytorch-job/pytorch-operator/overlays/application/application.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: app.k8s.io/v1beta1
```

Comment:
Does it make sense for the Application CR to be an overlay? Why not just create it by default?

Reply:
here are some of my thoughts of why we want the application overlay to be loosely bound.

1. It may vary by platform (I know we only offer a flavor of an application right now)
2. Different users may feel different 'applications' involve several manifests
3. They may be application-controllers that provide different behaviors
4. It's not required for the component to function (and could be viewed similarly as logging or istio)